src/ui/FacetSlider/FacetSlider.ts

```diff
@@ -40,52 +40,18 @@ export interface IFacetSliderOptions extends ISliderOptions {
 }
 
 /**
- * The FacetSlider component creates a facet containing a slider widget that allows the end user to filter results based
- * on a range of numerical values, rather than a "classic" multi-select {@link Facet} with a label and a count for each
- * value.
+ * The `FacetSlider` component creates a facet which contains a slider widget that allows the end user to filter results
+ * based on a range of numerical values (e.g., a date range, a price range, etc.).
  *
- * Note that this component does not inherit from the Facet component, and thus does not offer the same configuration
- * options. Also, some of the FacetSlider options cannot be set as a HTML attributes on the component and must rather be
- * configured in the {@link init} call of the search interface.
- *
```

Comment:
I removed these examples, because I feel the **Passing Component Options in the init Call** / **Passing Component Options Before the init Call** topics explain this pretty well, and the links are repeated in the options which can't be configured through the markup.